tests/k6/scenarios/core2_qa_control_plane_ops.js

```diff
@@ -127,8 +143,27 @@ function handleCtlOp(config, op, modelTypeIx, existingModels) {
                     }
                 }
                 modelCRYaml = yamlDump(newModelCR)
-            } catch (_) {
+                if (config.isLoadPipeline) {
+                    let pipeline = kubeclient.get(seldonObjectType.PIPELINE.description, generatePipelineName(modelName), config.namespace)
+                    let steps = pipeline.spec.steps
+                    steps[0]["batch"] = {"size": Math.round(Math.random() * 100)}  // to induce a change in pipeline
+                    let newPipelineCRYaml = {
+                        "apiVersion": "mlops.seldon.io/v1alpha1",
+                        "kind": "Pipeline",
+                        "metadata": {
+                            "name": generatePipelineName(modelName),
+                            "namespace": getConfig().namespace,
+                        },
+                        "spec": {
+                            "steps": steps,
+                            "output": pipeline.spec.output
+                        }
+                    }
+                    pipelineCRYaml = yamlDump(newPipelineCRYaml)
+                }
+            } catch (err) {
                 // just continue test, another VU might have deleted the chosen model
+                console.log(`Failed to update model ${modelName}: ${err}`)
                 return false
```

Comment:
Just a note that I need to change this return in my PR (now this function returns a seldonOpExecStatus rather than bool)

Reply:
I can make that change in my follow-up PR though